scripts/ci.sh

```diff
@@ -16,8 +16,9 @@ WITH_REPORT_SERVER=true
 WITH_INFRA_PRODUCTION=true
 WITH_INFRA_STAGING=true
 WITH_BENCHMARK=true
+WITH_BUILDINFRA_POC=true
```

Comment:
Lets not touch the ci.sh test script for this. We can move this into INFRA_PRODUCTION if we decide we want to go this route.